index.js

```diff
@@ -8,7 +8,10 @@ const exclusions = ['--help'];
 
 module.exports = function (cb) {
   try {
-    cb(null, require(tmpfile));
+    var flags = require(tmpfile);
```

Comment:
@tkellen you can't try/catch around process.nextTick, so I am assigning the require to a variable sync but calling the callback in the process.nextTick to avoid zalgo